cli.js

```diff
@@ -78,8 +81,34 @@ Usage: $0 <command> [options]` // eslint-disable-line comma-dangle
         describe: 'Use directory dir. "create" will create this directory if needed.',
         type: 'string',
       })
-      .strict();
+      .strict()
+      .example(
+` // Create a function from python3 template
+  bn create python3 ninja
+
+  // Create a function from node8 template with concurrent execution model
+  bn create node8 pirate --config.executionModel=concurrent
+
+  // Create a function from python2 template with exclusive execution model and FOO env
+  bn create python2 hello --config.executionModel=exclusive --config.env.FOO=bar
+`);
   }, async (argv) => {
+    // validating here and not in coerce to keep output consistent
+    if (argv.config !== undefined) {
+      const configType = typeof argv.config;
+      if (configType !== 'object' || Array.isArray(argv.config)) {
+        msgAndExit(`Non object create configuration options: ${argv.config}`, true);
+      }
+      // transform boolean true  for --config.env.VAR to work properly
+      if (argv.config.env && typeof argv.config.env === 'object') {
```

Comment:
1. This doesn't guard against `bn create python2 hello --config.env.FOO.BOO=bar`, it actually creates the broken `binaris.yml`.

Reply:
indeed. it doesn't do any value sanitization on purpose. it's simply a mechanism to modify the yaml at create time. a user can create an invalid configuration, just as is possible with an editor.

Reply:
Hmm not sure I see eye to eye here. That being said, if its the behavior you were aiming for, it's not block worthy.